### DISCUSSION TITLE
[FEATURE] Add a challenge for basic XSS

### DISCUSSION THREAD

**Discussion by ShreeluSantosh** [lennis-dev/hackit#19] [Ideas]
### Is there an existing discussion for this?

- [X] I have searched the existing discussions

### Describe your idea

I was wondering about a challenge to make players aware of the risks of XSS in web pages. Here's a fun challenge for this:

Name: A Recipe for Disaster
Description: Submit your favorite recipe with a little surprise in it! Hint: You might want to spice it up with a bit of JavaScript :)

**Comment by lennis-dev** [comment]
I have created an issue #20, should I assign you?

**Reply by ShreeluSantosh** [reply]
Yes
